distribution/docs/src/main/resources/draft-documentation.adoc

```diff
@@ -574,6 +574,8 @@ include::{adoc-include}/_plugins/access-plugin-intro-contents.adoc[]
 include::{adoc-include}/_plugins/access-plugins-contents.adoc[]
 
 include::{adoc-include}/_plugins/operation-plugin-contents.adoc[]
+
+include::{adoc-include}/_plugins/security-audit-plugin-contents.adoc[]
```

Comment:
the `{adoc-include}` will need to be `{adoc-include-cal}` to find files in Alliance.